### PR TITLE
Collar, Sleeve, Style features added.

### PR DESCRIPTION
**Pull request by danille**

### CODE REVIEW

File: app/src/test/java/io/github/wzieba/vericco/Model/CollarFeatureTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    @Before
+    public void setUp() {
+        collarFeature = new CollarFeature();
+        collarFeatureItems = collarFeature.getItems();
+    }
+
+
+    /*
+     * NOTE: Should we test CollarEnum first, and only after it test Collar Constructor?
```
Comment (wzieba) [inline app/src/test/java/io/github/wzieba/vericco/Model/CollarFeatureTest.java]: Ok we can think about refactoring our tests. For now, please remove comment and empty lines above

File: app/src/main/java/io/github/wzieba/vericco/Model/StyleFeature.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+public class StyleFeature extends Feature {
+
+    public StyleFeature() {
+        super("Style", StyleEnum.values());
+    }
+
+    private enum StyleEnum implements FeatureItemInterface {
+        SLIM("Slim", R.drawable.ic_android_black_24dp, 10f),
+        REGULAR("Regular", R.drawable.ic_android_black_24dp, 10f),
+        RELAXED("Relaxed", R.drawable.ic_android_black_24dp, 10f);
```
Comment (wzieba) [inline app/src/main/java/io/github/wzieba/vericco/Model/StyleFeature.java]: I think if we have got base price that prices of style should be 0. What do you think?

File: app/src/main/java/io/github/wzieba/vericco/Model/CollarFeature.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+public class CollarFeature extends Feature {
+    public CollarFeature() {
+        super("Collar", CollarEnum.values());
+    }
+
+    private enum CollarEnum implements FeatureItemInterface {
+        BUSINESS("Business", R.drawable.ic_android_black_24dp, 10f),
+        CUTAWAY("Cutaway", R.drawable.ic_android_black_24dp, 10f),
+        MAO("Mao", R.drawable.ic_android_black_24dp, 8f);
```
Comment (wzieba) [inline app/src/main/java/io/github/wzieba/vericco/Model/CollarFeature.java]: Please note, that due to our idea price of one element in every Feature should be 0 - it's like from "base product"

File: app/src/main/java/io/github/wzieba/vericco/Model/SleeveFeature.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
+package io.github.wzieba.vericco.Model;
+
+import io.github.wzieba.vericco.R;
+
+/**
+ * Created by danille on 5/14/17.
+ */
```
Comment (wzieba) [inline app/src/main/java/io/github/wzieba/vericco/Model/SleeveFeature.java]: Remove those comments

File: app/src/test/java/io/github/wzieba/vericco/Model/CollarFeatureTest.java [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+import org.junit.Before;
+import org.junit.Test;
+
+import io.github.wzieba.vericco.R;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by danille on 5/14/17.
+ */
```
Comment (wzieba) [inline app/src/test/java/io/github/wzieba/vericco/Model/CollarFeatureTest.java]: The same as above

File: app/src/main/java/io/github/wzieba/vericco/Model/StyleFeature.java (hunk 2 above)
Comment (danille) [inline app/src/main/java/io/github/wzieba/vericco/Model/StyleFeature.java]: ok, you're right with this.

**Pull request merged**